### PR TITLE
KOGITO-2464 Process list show load more buttons when no results found

### PR DESCRIPTION
**Pull request by AjayJagan**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
Description : This pr fixes a bug which showed loadmore when there were no instances present in the list
JIRA:  https://issues.redhat.com/browse/KOGITO-2464

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@AjayJagan can we add some test please?

File: ui-packages/packages/management-console/src/components/Templates/ProcessListPage/ProcessListPage.tsx [hunk 1]
```diff
@@ -344,7 +344,9 @@ const ProcessListPage: React.FC<InjectedOuiaProps> = ({ ouiaContext }) => {
               )}
               {(!loading || isLoadingMore) &&
                 !isLoading &&
-                initData !== undefined &&
+                initData &&
+                initData.ProcessInstances &&
+                initData.ProcessInstances.length !== 0 &&
```
Comment (nmirasch) [inline ui-packages/packages/management-console/src/components/Templates/ProcessListPage/ProcessListPage.tsx:349]: @AjayJagan @cristianonicolai  I think this condition is not enough consistent. With this condition in the first page when the results are less than page size the buttons are displayed anyway. but if you click load more and the limit is  less that page size the buttons disappear. For me, makes sense hide the buttons if results < page size (guessing there is nothing more to load ) at the first page too. Wdyt?
(I had to update the snapshot to run this code but maybe that would be covered in the test addition that @cristianonicolai commented)

Comment (cristianonicolai) [reply ui-packages/packages/management-console/src/components/Templates/ProcessListPage/ProcessListPage.tsx:349]: @nmirasch hm, makes sense to me.

**Comment by nmirasch** [comment]
@AjayJagan  what is missing here is to setLimit here https://github.com/kiegroup/kogito-apps/blob/72aec83704e7c9789ca5889b0a1e0d23ec606119/ui-packages/packages/management-console/src/components/Organisms/ProcessListTable/ProcessListTable.tsx#L71. That query don't update that limit used to know how much items were loaded last lime and used to know if the  load more buttons have to be shown. Let me try this to verify it covers all cases I'll let you know how it goes..

**Comment by AjayJagan** [comment]
> @AjayJagan  what is missing here is to setLimit here https://github.com/kiegroup/kogito-apps/blob/72aec83704e7c9789ca5889b0a1e0d23ec606119/ui-packages/packages/management-console/src/components/Organisms/ProcessListTable/ProcessListTable.tsx#L71. That query don't update that limit used to know how much items were loaded last lime and used to know if the  load more buttons have to be shown. Let me try this to verify it covers all cases I'll let you know how it goes.. 

Thanks a lot @nmirasch

**Comment by AjayJagan** [comment]
closing this PR as @nmirasch  has raised a new one for the same.

**Pull request closed without merging**